Packs/Spamcop/Integrations/Spamcop/Spamcop.py

```diff
@@ -0,0 +1,90 @@
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+
+''' IMPORTS '''
+
+
+import socket
+
+
+def results_return(command, thingtoreturn):
+    ip_reputation = {
+        'indicator': thingtoreturn['Address'],
+    }
+    try:
+        if thingtoreturn['Malicious']['Vendor']:
+            score = Common.DBotScore.BAD
+    except LookupError:
+        score = Common.DBotScore.NONE
+    dbot_score = Common.DBotScore(
```

Comment:
You also use the readable_output entry and return some message to the user

Reply:
lets leave some improvement for v2 ;)